account_invoice_merge_operating_unit/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# Copyright 2017 Serpent Consulting Services Pvt. Ltd.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+{
+    'name': "Account Invoice Merge Operating Unit",
+    'author': "Eficent Business and IT Consulting Services S.L., "
+              "Serpent Consulting Services Pvt. Ltd.,"
+              "Odoo Community Association (OCA)",
+    'website': "http://www.eficent.com",
+    'category': 'Finance',
+    'version': '10.0.1.0.0',
+    'license': 'AGPL-3',
+    'depends': [
+        'account_invoice_merge',
```

Comment:
Please add a oca-dependency with `account-invoicing` in order to be able to build correctly the runbot and execute the tests.